src/Components/Nav/CommunityNav/CommunityNav.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import "./CommunityNav.scss";
+
+class CommunityNav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      display: "flex",
+      nonMemberDisplay: "flex",
+      memberDisplay: "none",
+    };
```

Comment:
현재 어떤 상황에 따라 display none과 flex를 위해 상태를 만드셨네요!
하지만, 이런 상황에서는 css로 state에 따라 숨기고 처리하는 방법보다 **조건부 렌더링**을 사용하는 것이 더 좋습니다.
현재 3가지 state가 있는데 변수명도 아래와 같이 더 직관적으로 변경하면 좋을 것 같네요 :)
```suggestion
    this.state = {
      display: true,
      displayNonMemeber: true,
      displayMember: false,
    };
```

Reply:
네 알겠습니다!